packages/workspace/src/tasks-runner/empty-reporter.ts

```diff
@@ -1,6 +1,9 @@
 import { Task } from '@nrwl/devkit';
 import { Reporter, ReporterArgs } from './reporter';
 
+/**
+ * @deprecated the reporter API is no longer supported
```

Comment:
Same minor suggestion about specific reference to `v14` if we already know that will be the version it will be removed in